### PR TITLE
A few localization fixes

### PR DESCRIPTION
**Pull request by marusak**

### CODE REVIEW

File: pkg/lib/machine-info.js [hunk 1]
```diff
@@ -243,7 +243,7 @@ function processMemory(info) {
         let memorySize = memoryProperty.Size || _("Unknown");
         if (memorySize.includes("MB")) {
             const memorySizeValue = parseInt(memorySize, 10);
-            memorySize = memorySizeValue / 1024 + " GB";
+            memorySize = cockpit.format(_("$0 GB"), memorySizeValue / 1024);
```
[the remark this answers is not in the document]
Comment (marusak) [reply pkg/lib/machine-info.js]: As discussed on IRC, for example Russian translates `GiB` so let's translate it. I changed it to `GiB` though.

**Review by martinpitt — DISMISSED** [review]
Danke!

**Comment by martinpitt** [comment]
Argh, [tests need updating](https://logs.cockpit-project.org/logs/pull-16538-20211102-061708-64fb8509-ubuntu-2004/log.html#293-2) for the GB → GiB renaming

**Comment by marusak** [comment]
Updated

**Review by martinpitt — APPROVED** [review]

**Comment by marusak** [comment]
[polkitd](https://logs.cockpit-project.org/logs/pull-16538-20211102-093808-427b29b1-fedora-35/log.html#291-2) is dumping core. This is surely not related.

**Comment by martinpitt** [comment]
@marusak : It's not related. This is https://github.com/cockpit-project/bots/issues/2567 , I was hoping it was fixed on current OSes, but seems not.. so we need to copy the naughty.

**Comment by martinpitt** [comment]
I sent https://github.com/cockpit-project/bots/pull/2585 to copy the naughties, which addresses the F35/ubuntu-stable failures.  Let's retry after that landed.

The [mobile pixel test failure](https://logs.cockpit-project.org/logs/pull-16538-20211102-104210-427b29b1-fedora-34-mobile/pixeldiff.html#TestKeys-testPrivateKeys-ssh-keys-dialog-mobile) has also plagued us for a while now, also unrelated.

**Review by martinpitt — APPROVED** [review]
I pushed a no-change rebase to pick up the recent test fixes and fix packit.

**Pull request merged**